Update ex3.html

Yuriy2437 opened the pull request:
Вместо кликабельных ссылок на интернет-сайты поставлены кликабельные ссылки на слова Телеграм, YouTube, Instagram, как более соответствующие условию задачи

Yuriy2437 committed:
Update ex3.html

Вместо кликабельных ссылок на интернет-сайты поставлены кликабельные ссылки на слова Телеграм, YouTube, Instagram, как более соответствующие условию задачи

foxprogs commented:
@Yuriy2437 надо переименовать ПР во что-то типа "исправление по заданию HTML"

foxprogs commented:
@Yuriy2437 во втором задании по html последний HTML должен быть жирный 
![изображение](https://github.com/tt-dev-23/CV_Yuriy/assets/30548361/ee85c650-ce3d-4a35-b29e-780230938819)

foxprogs commented:
@Yuriy2437 в четвертом задании нужны картинки, а там ссылки. Надо исправить, задание не выполнено. 
![изображение](https://github.com/tt-dev-23/CV_Yuriy/assets/30548361/147a1173-2fac-47e9-a1b4-c1784527f91f)

foxprogs commented:
@Yuriy2437 6 задание
радиобаттон не с той стороны, должен быть слева а не справа
поле с почтой сверстано не так как все остальные, надо в одном стиле
поле с датой сделано тоже не по заданию и там ошибка в верстке. Должно было подсветить что не закрыт тег
При отправке ничего не передается, надо исправить чтобы передавались данные с полей и радиобаттон

foxprogs commented:
![изображение](https://github.com/tt-dev-23/CV_Yuriy/assets/30548361/8692ad96-c9d3-4f7f-8977-da7ea3db1511)
При отправке только какой-то question=no передается
@Yuriy2437

Yuriy2437 committed:
Create index.html

Yuriy2437 committed:
Create css_practice

Yuriy2437 committed:
Delete css_practice

Yuriy2437 committed:
Create index.html

Yuriy2437 committed:
Delete css_practice directory

Yuriy2437 committed:
Delete cv_css directory

Yuriy2437 committed:
Create index.html

Yuriy2437 committed:
Create index.html

Задания 1-6 по курсу CSS

Yuriy2437 committed:
Add files via upload

CV (визитка) по курсу CSS

Yuriy2437 committed:
Create index.html

Задание 1 по курсу CSS

Yuriy2437 committed:
Delete css_practice/ex1 directory

Yuriy2437 committed:
Delete css_practice directory

Yuriy2437 committed:
Create readme.md

Папка для заданий 1-6 по курсу CSS

Yuriy2437 committed:
Create readme.md

Задание 1 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Create readme.md

Папка для задания 2 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Create readme.md

Папка для задания 3 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Create readme.md

Папка для задания 4 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Create readme.md

Папка для задания 5 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Create readme.md

Папка для задания 6 по курсу CSS

Yuriy2437 committed:
Add files via upload

Yuriy2437 committed:
Update README.md

Исправлено название задания

Yuriy2437 committed:
Add files via upload

Во втором задании по HTML последний "HTML" выделен жирным шрифтом

Yuriy2437 committed:
Add files via upload

В четвертом задании вместо ссылок выведены картинки

Yuriy2437 committed:
Add files via upload

радиобаттон смещен на правильную сторону, 
все поля сделаны в одном стиле,
исправлена ошибка в поле с датой,
исправлена отправка: теперь передаются данные с полей и радиобаттон

Yuriy2437 committed:
Add files via upload

убран пустой стиль body, исправлена невалидная верстка, соблюдена структура: тег р на разных уровнях с div

Yuriy2437 committed:
Add files via upload

табы заменены на пробелы

Yuriy2437 committed:
Add files via upload

в файле index в классе flag теперь два пробела, а не четыре,
в файле css убрана пустая строка

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Add files via upload

переделана структура полностью, 
шрифт без засечек, скругления, отступы,
убрана лишняя линия

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Add files via upload

Полностью переделана структура на основе переделанного задания 4,
Исправлен hover, чтобы выделенный текст стал белым

Yuriy2437 committed:
Delete readme.md

Yuriy2437 committed:
Add files via upload

Переделаны рамка, скругления, шрифты, отступы. 
Сделано pixel-perfect, чтобы было максимально так же, как на картинке.

Yuriy2437 committed:
Add files via upload

Убран лишний пустой код с классами в index.html,
В styles.css убраны лишняя точка с запятой, закомментированные стили, пробел перед px,
В мобильной версии убран отступ слева, чтобы изображение было по центру

Yuriy2437 committed:
Delete css_practice directory

Yuriy2437 committed:
Delete cv_css directory

Yuriy2437 committed:
Delete BESPECHANSKY.jpg

Yuriy2437 committed:
Delete ex1.html

Yuriy2437 committed:
Delete ex2.html

Yuriy2437 committed:
Delete ex3.html

Yuriy2437 committed:
Delete ex4.html

Yuriy2437 committed:
Delete ex5.html

Yuriy2437 committed:
Delete README.md

Yuriy2437 committed:
Delete ex6.html

Yuriy2437 committed:
Delete index.html

Yuriy2437 committed:
Delete reference.html
